src/toil/jobStores/aws/jobStore.py

```diff
@@ -299,6 +303,65 @@ def deleteFile(self, jobStoreFileID):
         else:
             info.delete()
 
+    def importFile(self, sourceUrl):
+        url = urlparse.urlparse(sourceUrl)
+        if url.scheme.lower() == 's3':
+            if len(sourceUrl.split('/')) != 4:
```

Comment:
It would be better to use a regex to check if the URL is syntactically valid in the actual if statements e.g. if re.match(<s3 pattern>, url) is not None: ... elif re.match(<s3 pattern>, url) is not None: ... else: runtime error